main.tf

```diff
@@ -243,8 +237,7 @@ resource "aws_security_group" "lambda_sg" {
 }
 
 resource "aws_lambda_function" "api_lambda" {
-  filename         = data.archive_file.cleanup_lambda_zip.output_path
-  source_code_hash = data.archive_file.cleanup_lambda_zip.output_base64sha256
```

Comment:
Can you add this back in with
```
  source_code_hash = filebase64sha256(var.lambda_zip_file)
```

Reply:
This is important so that TF compares the hashes and only pushes a new version of the lambda if there is actually a change.

Reply:
Ah, that's what it did! Done.